gematria/datasets/pipelines/benchmark_bbs.py

```diff
@@ -30,6 +31,15 @@
 _OUTPUT_FILE_PATTERN = flags.DEFINE_string(
     'output_file_pattern', None, 'The output file path/pattern.', required=True
 )
+_BENCHMARK_SCHEDULER = flags.DEFINE_enum(
```

Comment:
Abseil has `flags.DEFINE_enum_class()` that allows defining flags from Python enums without any extra boilerplate.